build/Dockerfile.build

```
@@ -0,0 +1,10 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal as builder
+
+RUN microdnf install -y make golang; microdnf -y clean all
+
+FROM builder
+
+RUN mkdir -p /output
+COPY . /ocs-operator
+WORKDIR /ocs-operator
+RUN make deps-update && make go-build OUTDIR_BIN=/output
```

```suggestion
RUN make build-go OUTDIR_BIN=/output
```
since build-go already calls deps-update

---

Legit.